### PR TITLE
Mid-Project PR

### PR DESCRIPTION
**Pull request by ElliotGardner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,3 +65,11 @@
 * New Features - Days Until Sell Out Prediction
 * New Features - User Event Watchlist
 * New Features - Customized User Event Digest
+
+Instructions:
```
Comment (aburzinski) [inline README.md:69]: I would try to call out this section more, so it is easier for users to find.  In addition, it should be written so more generally.  How does a user run this app if they don't have an API key?  How do they create their own?

In addition, you should include instructions for setting up the python environment.  I would recommend virtualenv over conda.  The instructions can be similar to:
- run virtualenv -p python3 my_proj
- run source my_proj/bin/activate
- run pip install -r requirements.txt

File: config/config.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+ingest_data:
```
Comment (aburzinski) [inline config/config.yml:1]: Will the user need to change this file at all?

**Comment by aburzinski** [comment]
Functionality
- The app functions as it should.  The ingest script successfully pulls data from the Eventbrite API and saves it to S3
- The create database script creates the database successfully assuming the user enters valid location/connection string information
- It may be worth while to spend some time fixing bugs related to if the user enters invalid connection string information (i.e. if the sqlite path they enter does not exist)

Readability and Code Quality
- It may help users understand your code more if you add longer doc strings  to some of your functions.  For example, one the functions in your create_database.py file is just """runs the creation script""".  It might be better to discuss where the database is created or how this function uses the config.yml file and user inputs
- The doc strings in the ingest_data.py file look very good
- There are good/helpful comments throughout code
- The names of variables and functions look good (they are descriptive)
- The ingest_data.py file is very long (300+ lines).  Does it make sense to move any of the helper functions to a different script and import it?

Overall Design
- I like the design decision of running a batch script twice daily and storing the results.  This gives you past data to work with, which the Eventbrite API does not provide (as it is only forward looking)
- I also like the decision to run the batch process twice daily as opposed to once.  This gives you better granularity, and since the batch process doesn't take long to run and the pulled data isn't large, it is a worthwhile trade-off

Testing
- I noticed you aren't testing any of your code.  I added some pytest scripts to my project for some of the simpler functions.  These scripts helped me find issues that occur when an empty value is input, etc
- Some recommendations for tests in your project are:
	- Make sure files are created in S3 when you expect them to be
	- Make sure the format of the data from the Eventbrite API is in the format you expect
	- You could create helper functions to create the filenames of the files you are sending to S3, and test those helper functions to make sure they give you the correct names
	- Etc

Documentation
- It may help users understand your code more if you add several things to the readme
- You should include how to install the requirements.txt file
- You should also include any environment variables that need to be added (for example your RDS connection info)
- I would also give a short overview about what is included in the config/config.yml file (especially if the user needs to change any of these values when running your app)
- Good commit messages

**Comment by aburzinski** [comment]
Adding issues I opened to the PR so Chloe and Fausto can see them:

https://github.com/ElliotGardner/sell_out_project/issues/2

https://github.com/ElliotGardner/sell_out_project/issues/3

File: config/config.yml (hunk 2 above)
Comment (ElliotGardner) [reply config/config.yml:1]: The user won't have to, these can serve as the defaults, but they can be changed to adjust the behavior of some scripts (where to save the data, how to create the database, etc.).

File: README.md (hunk 1 above)
Comment (ElliotGardner) [reply README.md:69]: That makes sense, I will incorporate that into the next sprint.

**Comment by aburzinski** [comment]
Changes based on my feedback look good.

**Pull request merged**